### DISCUSSION TITLE
Version 3.0.x.x_Maintenance: increase client-side security by updating jquery & bootstrap

### DISCUSSION THREAD

**Discussion by devincentiis** [opencart/opencart#12305] [Ideas]
I propose to increase client side security by updating jquery to version 3.6.1 and bootstrap 3.4.1. Doing these two substitutions will no longer work datetimerpicker but just edit \catalog\view\javascript\jquery\datetimepickerbootstrap-datetimepicker.min.js replacing ".size()" with ".length" as indicated here: [https://stackoverflow.com/questions/412 ... e-jquery-3](https://stackoverflow.com/questions/41236574/bootstrap-datetimepicker-not-working-with-the-jquery-3) that everything will work again.
The same should be done on the backend.
Thanks for the attention.

**Comment by mhcwebdesign** [comment]
I have now upgraded the jquery to version 3.7.0, and also fixed the bootstrap-datetimepicker. Also, the Klarna Checkout payment extension has been removed, it uses the old and retired vendor library klarna/kco_rest. 

Next I am looking into bootstrap upgrade, and some upgrades of the 3rd party vendor libraries.

Update: Bootstrap has now been upgraded to version 3.4.1.
